pkg/controller/account/ec2.go

```diff
@@ -346,13 +346,13 @@ func (r *ReconcileAccount) getDesiredVCPUValue(reqLogger logr.Logger) (float64,
 		err = awsv1alpha1.ErrInvalidConfigMap
 	}
 	if err != nil {
-		reqLogger.Error(err, "failed getting desired vCPU quota from configmap data")
+		reqLogger.Info(fmt.Sprintf("Failed getting desired vCPU quota from configmap data, defaulting quota to: %v", 0))
```

Comment:
Minor: No need to Sprintf if you know we're defaulting to 0, can just log the string